### PR TITLE
Feature/log service

### PR DESCRIPTION
**Pull request by KazdenJolley**
### Log a Service Form

**Invited Collaborator(s) to Review
- Anthony, Jaden, and John

**What work was done?**
- I updated the AddServiceForm component so that it would send an object appropriate to what the backend was expecting

**Why was this work done?**
- This is an essential feature of the application, to be able to log services

**What feature / user story is it for?**
[Trello Card](https://trello.com/c/48AmopXM)

### Type of change
- Updated feature (non-breaking change which adds functionality)

Testing
- Has this feature been tested before conflict? Yes
- Has this feature been tested after conflict? Yes

Change Status
- Completed, ready to review and merge

### Checklist
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- My code has been reviewed by at least one peer
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- There are no merge conflicts

[Loom Video](https://www.loom.com/share/b774d6fb4ed64358a01955761b12c69b)

### CODE REVIEW

File: src/components/forms/AddServiceForm.js [hunk 1]
```diff
@@ -33,7 +43,6 @@ function AddServiceForm({
           form
             .validateFields()
             .then(values => {
-              console.log('values', values);
```
Comment (john-laubscher) [inline src/components/forms/AddServiceForm.js:36]: Good catch on the console.log!

**Review by john-laubscher — APPROVED** [review]
Looks good! Great job getting all the pieces together to fix the adding a service.

File: src/components/forms/AddServiceForm.js [hunk 2]
```diff
@@ -157,12 +162,12 @@ function AddServiceForm({
             ]}
           >
             <Select size="large" placeholder="Select Service Type">
-              {serviceTypes.map(item => (
+              {serviceTypePrograms.map(item => (
                 <Select.Option
```
Comment (Anthony-Cortese) [inline src/components/forms/AddServiceForm.js:166]: Great work implementing the serviceTypePrograms into the form!

File: src/components/pages/Services/RenderServicesPage.js [hunk 3]
```diff
@@ -61,8 +61,24 @@ function RenderServicesPage({
...
+    const service_time = provided_at.format('HH:mm:ss');
+    const service_duration = duration.format('HH:mm:ss');
+    const service_entry_data = {};
+
+    const newService = {
+      ...values,
+      service_date,
+      service_time,
+      service_duration,
+      service_entry_data,
```
Comment (Anthony-Cortese) [inline src/components/pages/Services/RenderServicesPage.js:75]: Great update to the onCreate function!

**Review by Anthony-Cortese — APPROVED** [review]
These changes look great, I was very happy to see the correlation between the service type and the programs, and the Stakeholder will be happy with this implementation! Great work Kazden!

**Review by Anthony-Cortese — APPROVED** [review]
These changes look great, I was very happy to see the correlation between the service type and the programs, and the Stakeholder will be happy with this implementation! Great work Kazden!

**Pull request merged**